bin/tachyon-workers.sh

```diff
@@ -15,7 +15,7 @@ if [ $# -le 0 ]; then
 fi
 
 bin=`cd "$( dirname "$0" )"; pwd`
-DEFAULT_LIBEXEC_DIR="$bin"/../libexec
+DEFAULT_LIBEXEC_DIR=`echo $bin | sed "s/bin$/libexec/g"`
 TACHYON_LIBEXEC_DIR=${TACHYON_LIBEXEC_DIR:-$DEFAULT_LIBEXEC_DIR}
 . $TACHYON_LIBEXEC_DIR/tachyon-config.sh
 
```

Comment:
If you guys prefer, I can make it one command, removing $bin variable as below:

DEFAULT_LIBEXEC_DIR=`cd "$( dirname "$0" )"; pwd | sed "s/bin$/libexec/g"`

Regards.